### PR TITLE
add negative test case test_searchedsorted.py

### PR DESCRIPTION
**Pull request by XiaLuNV**

### CODE REVIEW

File: tests/integration/test_searchsorted.py [hunk 1]
```diff
@@ -176,6 +176,52 @@ def test_ndim(ndim, side):
     check_api(a, None, v, side)
 
 
+def test_ndim_positive():
```
Comment (bryevdv) [inline tests/integration/test_searchsorted.py:179]: The name of this test is very confusing. I would suggest collecting all the error-checking tests into a class `Text_searchsorted_Errors` (or whatever, similar to other recent PRs) and then this coud be `test_ndim_mismatch` or something  like that.

File: tests/integration/test_searchsorted.py [hunk 2]
```diff
@@ -176,6 +176,52 @@ def test_ndim(ndim, side):
...
+    a_argsorted = np.random.random((5, 5, 5))
+    expected_exc = ValueError
+    with pytest.raises(expected_exc):
+        num.searchsorted(a, v, sorter=a_argsorted)
+    with pytest.raises(expected_exc):
+        # Numpy raises TypeError
+        np.searchsorted(a, v, sorter=a_argsorted)
+
+
+def test_sorter_shape():
```
Comment (bryevdv) [inline tests/integration/test_searchsorted.py:201]: size mismatch, not shape, no?

File: tests/integration/test_searchsorted.py (hunk 1 above)
Comment (XiaLuNV) [reply tests/integration/test_searchsorted.py]: fix

File: tests/integration/test_searchsorted.py (hunk 2 above)
Comment (XiaLuNV) [reply tests/integration/test_searchsorted.py]: fix

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
LGTM (I don't think the test failures are related to this PR)

**Pull request merged**